### PR TITLE
Bug 1756141: Remove incorrect optional tag from new ingress field

### PR DESCRIPTION
**Pull request by ironcladlou**
`loadBalancer` is a new field that contains `scope`, and `loadBalancer` is
optional (correct). However, `loadBalancer.scope` _should_ be required. This
commit fixes that.

Backport of https://github.com/openshift/api/pull/446

### CODE REVIEW

**Comment by ironcladlou** [comment]
@deads2k @sttts not sure what's up with the protobuf changes that got emitted here. If they shouldn't have happened I can manually delete them for now rather than investigate why the local tooling produced the changes.

File: quota/v1/generated.pb.go [hunk 1]
```diff
@@ -19,23 +19,18 @@
...
-	strings "strings"
-
-	reflect "reflect"
-
-	io "io"
-)
+import io "io"
 
 // Reference imports to suppress errors if they are not otherwise used.
 var _ = proto.Marshal
```
Comment (sttts) [inline quota/v1/generated.pb.go]: these are many pb changes

Comment (ironcladlou) [reply quota/v1/generated.pb.go]: I manually deleted these changes from the commit, not sure what's up with the generator. Will look into it later.

**Comment by ironcladlou** [comment]
@smarterclayton PTAL

**Comment by eparis** [comment]
I'm on my phone. Ben please go ahead and set cherry pick approved valid bug and unset invalid book

**Pull request merged**